### PR TITLE
Fix markdown block quote truncation

### PR DESCRIPTION
**Pull request by RobertSzefler**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -71,6 +71,8 @@ botocore = "1.31.72"
 boto3 = "1.28.72"
 requests = "^2.32.3"
 certifi = "^2023.7.22"
+regex = "2024.5.15"
```
Comment (RobertSzefler) [inline pyproject.toml:74]: This because stdlib `re` does not support reverse search (`regex.REVERSE`) up to Python 3.12

File: pyproject.toml [hunk 2]
```diff
@@ -71,6 +71,8 @@ botocore = "1.31.72"
...
+more_itertools = { version = "^10.3", python = "<3.12" }
```
Comment (RobertSzefler) [inline pyproject.toml:75]: This because `batched` appeared in stdlib `itertools` only in Python 3.12

File: src/robusta/core/sinks/transformer.py [hunk 3]
```diff
@@ -58,6 +58,10 @@ def apply_length_limit(msg: str, max_length: int, truncator: Optional[str] = Non
         truncator = truncator or "..."
         return msg[: max_length - len(truncator)] + truncator
 
+    @staticmethod
+    def apply_length_limit_to_markdown(msg: str, max_length: int, truncator: str = "...") -> str:
+        return trim_markdown(msg, max_length, truncator)
```
Comment (Avi-Robusta) [inline src/robusta/core/sinks/transformer.py:63]: i would put this in a try catch and if this fails try to run apply_length_limit

**Review by Avi-Robusta — APPROVED** [review]
LGTM one comment

File: src/robusta/core/sinks/transformer.py (hunk 3 above)
Comment (RobertSzefler) [reply src/robusta/core/sinks/transformer.py]: done

**Pull request merged**